eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,4 +1,4 @@
-<!-- Whenever altering this or other Source Build files, please include @dotnet/source-build-internal as a reviewer. -->
+<!-- When altering this file or making other Source Build related changes, include @dotnet/source-build as a reviewer. -->
```

Comment:
Probably a dumb question. But now that we no longer have a workload for Aspire, do we need to be part of "source build" anymore?

cc @joperezr

Reply:
Currently the 8.2 aspire baseline manifest is shipping in 9.0 and present in 10.0. While that is the case, it must be part of the VMR. When that is removed, it can go away.

Reply:
> Currently the 8.2 aspire baseline manifest

is that still in the dotnet/aspire `main` branch? If so, can/should we remove it from dotnet/aspire `main` and the VMR can use it from the dotnet/aspire `release/8.2` branch?

Reply:
I propose I abandon this PR and open a new one that removes the sourcebuild.props and SourceBuildPrebuiltBaseline.xml files from main.  There is no value in updating the source-build team names in the 8.x branches.